Sources/GRPCCore/Call/Client/CallOptions.swift

```diff
@@ -77,6 +77,42 @@ public struct CallOptions: Sendable {
   /// reported to the client. Hedging is only suitable for idempotent RPCs.
   public var executionPolicy: RPCExecutionPolicy?
 
+  /// Message compression algorithms.
+  public struct CompressionAlgorithm: Sendable, Equatable {
+    private enum Value {
+      case useClientTransportDefault
+      case gzip
+      case deflate
+      case none
+    }
+
```

Comment:
Oh sorry, I didn't mean we should do this in this PR (or at all, I was just throwing out ideas for how we could improve it). For this PR we should just use `CompressionAlgorithm.none`.